metrics.md

```diff
@@ -0,0 +1,85 @@
+# Universal Search Metrics
```

Comment:
I agree this is a great start! I think we will end up reporting the engagement ratio for all Test Pilot tests "automatically". The metrics that I think are most important for this test specifically are the click through rates of the recommendations and retention. I'm working on a summary doc right now to capture these. We can integrate into this doc as soon as that's done.